taqueria-plugin-flextesa/package.json

```diff
@@ -36,13 +36,13 @@
     "directory": "taqueria-plugin-flextesa"
   },
   "dependencies": {
+    "@taqueria/node-sdk": "^0.0.8",
     "fast-glob": "^3.2.7",
-    "promise-retry": "^2.0.1",
-    "@taqueria/node-sdk": "^0.0.5"
+    "promise-retry": "^2.0.1"
   },
   "devDependencies": {
     "@types/promise-retry": "^1.1.3",
     "parcel": "^2.0.1",
-    "typescript": "^4.5.2"
+    "typescript": "4.5.4"
```

Comment:
Do we want to specifically lock in this version of typescript? Why are we getting rid of the ^?

Reply:
Hey Alex. Yep, that was deliberate. I mentioned this in a commit message but should have also mentioned it in the PR description - 4.5.5 has a bug: https://github.com/microsoft/TypeScript/issues/32408

Aside from that, as TypeScript is a core dependency, I think it does indeed make sense to lock down, as it's shared among many workspaces in the project. Without doing so, its possible that one workspace might use 4.5.6 and another use 4.5.7 (once those versions come out). For some dependencies, that probably doesn't matter but TypeScript seems more significant.